tests/ci/docker_images_helper.py

```diff
@@ -27,6 +30,22 @@ def docker_login(relogin: bool = True) -> None:
         )
 
 
+def ecr_login() -> str:
```

Comment:
do we need this functionality in oss?

Reply:
It doesn't harm, IMHO, and done in the matter of the least difference between CIs.

Reply:
Ok, valid. If we'll need it, then we can bring it from the cloud